action/execution.go

```diff
@@ -147,10 +147,10 @@ func (ex *Execution) ToJSON() (*explorer.Execution, error) {
 		Signature:      hex.EncodeToString(ex.signature),
 	}
 	if ex.amount != nil && len(ex.amount.Bytes()) > 0 {
-		execution.Amount = ex.amount.Int64()
+		execution.Amount = hex.EncodeToString(ex.amount.Bytes())
```

Comment:
@lizhefeng why do we use hex instead of itoa?

Reply:
@zjshen14 Do you mean use itoa to convert int to hex string?

Reply:
I mean e.g. 1234.5678 -> "1234.5678" instead of "0xA07E..."